server/sequencer_manager.go

```diff
@@ -66,6 +68,17 @@ func (s SequencerManager) ExecutePass(logIDs []int64, logctx LogOperationManager
 	toSeq := make(chan int64, len(logIDs))
 
 	for _, logID := range logIDs {
+		// Get a signer for this log, if there isn't one in s.signers already.
+		if _, ok := s.signers[logID]; !ok {
+			ctx := util.NewLogContext(logctx.ctx, logID)
+			signer, err := newSigner(ctx, s.registry, logID)
+			if err != nil {
+				glog.Errorf("Could not get signer for log %d: %v", logID, err)
+				continue
```

Comment:
Yes, continue is the right action, as it will prevent the logID being added to the `toSeq` channel, which avoids the null pointer scenario I think you're talking about. I'll add a test to confirm.